scripts/Jenkinsfiles/quality

```diff
@@ -74,7 +74,7 @@ pipeline {
         stage('Run Tests') {
             parallel {
                 stage("commonlib pylint") {
-                    agent { label "jenkins-worker" }
+                    agent { label "js-worker" }
```

Comment:
Can you add comments here?  Does `js-worker` just happen to have the amount RAM you want?  Without a comment, it would be unclear why this agent is being used, and we also wouldn't know when to try going back down to lower RAM, for example, when a particular issue around caching is resolved (if that is possible), or when we have a lower threshold?